FV3/atmos_cubed_sphere/tools/coarse_grained_diagnostics.F90

```diff
@@ -739,28 +740,15 @@ subroutine fv_coarse_diag(Atm, Time)
 
     if (need_height_array) then
       allocate(height_on_interfaces(is:ie,js:je,1:npz+1))
-      if(Atm(tile_count)%flagstruct%hydrostatic) then
-        call compute_height_on_interfaces_hydrostatic( &
-          is, &
-          ie, &
-          js, &
-          je, &
-          npz, &
-          Atm(tile_count)%pt(is:ie,js:je,1:npz), &
-          Atm(tile_count)%peln(is:ie,1:npz+1,js:je), &
-          height_on_interfaces(is:ie,js:je,1:npz) &
-        )
-      else
-        call compute_height_on_interfaces_nonhydrostatic( &
-          is, &
-          ie, &
-          js, &
-          je, &
-          npz, &
-          Atm(tile_count)%delz(is:ie,js:je,1:npz), &
-          height_on_interfaces(is:ie,js:je,1:npz) &
-        )
-      endif
+      call get_height_field(is, ie, js, je, Atm(tile_count)%ng, npz, &
```

Comment:
This isn't something we wrote but is available?

Reply:
That's right...I'm a little embarrassed I didn't use it all along. https://github.com/VulcanClimateModeling/fv3gfs-fortran/blob/d6f52a8f6601ba64b303a54429ba315eeb495b4f/FV3/atmos_cubed_sphere/tools/fv_diagnostics.F90#L3341-L3381